Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 1.1.0-develop
+version: 1.1.1-develop
```

Comment:
This doesn't need to be bumped since we don't build develop versions, it can be left alone until we make a release